app/models/otp_request.rb

```diff
@@ -8,4 +8,8 @@ class OtpRequest < ApplicationRecord
 
   validates :otp, presence: true, numericality: { only_integer: true }, length: { is: OTP_LENGTH }
   validates :mobile_number, format: { with: VALID_MOBILE_NUMBER_REGEX }, length: { is: MOBILE_NUMBER_LENGTH }
+
+  def expired?
+    expire_at && expire_at < Time.now
```

Comment:
This nil check is because we had missed adding non-null constraints on relevant columns of otp_requests table. I've a separate [ticket ](https://vijithmv.atlassian.net/browse/PLM-30)to fix this issue. This check will be removed as part of PLM-30